anagram.py

```diff
@@ -9,44 +8,62 @@ def anagramchk(word,chkword):
             return 0
     return 1
 
+def hasAllRequired(word,required):
```

Comment:
Function names should ideally be lower cased and joined by underscore if need be- can you refactor this to `has_all_required`?